modules/nw-ovn-kubernetes-pod-connectivity-checks.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-pod-network-connectivity-checks_{context}"]
+= OVN-Kubernetes pod network connectivity checks
```

Comment:
This procedure looks like it's full of a lot of one-item procedure modules to me.

Reply:
@maxwelldb I think it is valid as they are all connectivity checks, I do think it would not look great if split into a lot of one-item procedure modules.